### PR TITLE
Make tag output comma-separated in CLI

### PR DESCRIPTION
**Pull request by a10y**
@percyliang

### CODE REVIEW

File: codalab/lib/bundle_cli.py [hunk 1]
```diff
@@ -1237,7 +1237,7 @@ def _worksheet_description(self, worksheet_info):
         fields = [
             ('Worksheet', self.worksheet_str(worksheet_info)),
             ('Title', formatting.verbose_contents_str(worksheet_info['title'])),
-            ('Tags', worksheet_info['tags']),
+            ('Tags', ', '.join(worksheet_info['tags'])),
```
Comment (percyliang) [inline codalab/lib/bundle_cli.py]: Minor: I think joining on a single space would be nicer.

Comment (a10y) [inline codalab/lib/bundle_cli.py]: Oh okay, is there no idea of multi-word tags in codalab?

Comment (percyliang) [inline codalab/lib/bundle_cli.py]: Yes (although we don't enforce it, but we should - add an issue?)  I think
it's pretty normal to have tags without spaces though.

Comment (a10y) [inline codalab/lib/bundle_cli.py]: Are you saying that keeping tags to single-words is not enforced? I can issue that

Comment (percyliang) [inline codalab/lib/bundle_cli.py]: Right

Comment (a10y) [inline codalab/lib/bundle_cli.py]: #336 Just registered

**Comment by percyliang** [comment]
LGTM

**Pull request merged**